service_discovery/core/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
```

Comment:
Please consider adding documentation to this folder.

Reply:
Agreed, I definitely will. I am weighing the option of including it in this PR or a follow-up. If I decide to do it in a follow-up, I will link a GitHub Issue for it to this comment

Reply:
Issue for adding documentation: https://github.com/eclipse-chariott/chariott/issues/145